### PR TITLE
Update democontent.md

### PR DESCRIPTION
**Pull request by alliomeria**
# Updated documentation

Updated links for OSX and Ubuntu links in Demo documentation

@DiegoPino Please review this. Thank you.

### CODE REVIEW

File: docs/osx.md [hunk 1]
```diff
@@ -11,7 +11,7 @@ and paste **a single line at a time** and press the `Enter` key afterwards. We s
 
 If something fails (and we hope it does not) troubleshooting will be much easier if you can share that output when asking for help.
 
-Happy deploying!
+Happy deploying!!
```
Comment (DiegoPino) [inline docs/osx.md]: Great. Good testing, now let's return this to a single `!`

**Review by DiegoPino — CHANGES_REQUESTED** [review]
Small change, reverting one of the test commits.

File: docs/osx.md (hunk 1 above)
Comment (alliomeria) [reply docs/osx.md]: Ok I fixed this

**Review by DiegoPino — APPROVED** [review]
Great! Merging and congratulations for your first Pull and Merge!

**Pull request merged**